conf/routes

```diff
@@ -14,10 +14,16 @@ GET     /vehicles/                  controllers.VehicleController.getVehicles
 
 GET     /users                      controllers.UserController.getFiltered(name:String)
 GET     /users/:id                  controllers.UserController.get(id:Int)
+
+GET     /users/:id/locations/       controllers.LocationController.getHistory(id:Long, from:java.sql.Timestamp, to:java.sql.Timestamp)
+
 POST    /users/                     controllers.UserController.create
 POST    /users/me/password          controllers.UserController.changeOwnPass
 
 POST    /location/                  controllers.LocationController.update
+POST    /location/history/          controllers.LocationController.postHistory
+GET     /location/                  controllers.LocationController.getOwnHistory(from:java.sql.Timestamp, to:java.sql.Timestamp)
```

Comment:
Wait, they're both `/locations/` with slashes (which is strange) in the end and they only differ with the type?
Umm
I don't like it, this may cause confusion

Reply:
Usually there are 4 routes for a single URL in REST approaches:
GET    /resource/ with url filter params - get a list of entities
POST /resource/ with body/form contents - create new entity
PUT   /resource/ with body/form - update
DELETE /resource/ with body/form or to /resource/<id> to delete entity 
It's more than ok

Reply:
Still don't understand slashes in the end
Why?

Reply:
Well, there's no specification for this, so I decided to roll with slashes everywhere. 
Why is it confusing?

Reply:
But they're not everywhere :/

We have
```
GET     /users
GET     /users/:id
POST    /users/me/password
```

and we have
```
GET     /users/:id/locations/ 
POST    /users/ 
POST    /location/ 
POST    /location/history/ 
GET     /location/
```
And it just blows my mind